kolibri/core/auth/management/commands/bulkexportusers.py

```diff
@@ -39,16 +40,34 @@
 # TODO: decide whether these should be internationalized
 labels = OrderedDict(
     (
-        ("id", _("Database ID ({})".format("UUID"))),
-        ("username", _("Username ({})".format("USERNAME"))),
-        ("password", _("Password ({})".format("PASSWORD"))),
-        ("full_name", _("Full name ({})".format("FULL_NAME"))),
-        ("kind", _("User type ({})").format("USER_TYPE")),
-        ("id_number", _("Identifier ({})".format("IDENTIFIER"))),
-        ("birth_year", _("Birth year ({})".format("BIRTH_YEAR"))),
-        ("gender", _("Gender ({})".format("GENDER"))),
-        ("enrolled", _("Learner enrollment ({})".format("ENROLLED_IN"))),
-        ("assigned", _("Coach assignment ({})".format("ASSIGNED_TO"))),
+        ("id", _("Database ID ({})").format("UUID")),
+        ("username", _("Username ({})").format("USERNAME")),
+        ("password", _("Password ({})").format("PASSWORD")),
+        ("full_name", _("Full name ({})").format("FULL_NAME")),
+        (
+            "kind",
+            pgettext_lazy(
+                "CSV header for the type of user: ADMIN, LEARNER, COACH...",
```

Comment:
I wonder if the wording `CSV column header for...` would sound more accurate...

Reply:
I wonder it too, but does it matter? this is for the translators to understand it.